src/stk/molecular/topology_graphs/cage/metal_topologies/m4l4_square.py

```diff
@@ -26,16 +26,31 @@ class M4L4Square(Cage):
 
     Examples
     --------
-
-    *Advanced Construction*
-
-    When building metal-organic structures, it is common that
-    functional groups, defined by metal atoms, will be overlapping.
-    Therefore, to build :class:`.M4L4Square` instances with the desired
-    building block alignment, corner and linker building blocks must
-    have functional groups with additional `placer` atoms. First we
-    build the metal atom (Pd(II) with four functional groups).
-
+    *Aligning Metal Complex Building Blocks*
+
+    When building metal-organic cages from metal complex 
+    building blocks, it is common that
+    the metal complex :class:`.BuildingBlock` will have 
+    multiple functional groups, but that those functional groups
+    are overlapping. This means that some of its atoms appear in 
+    multiple functional groups. A difficulty arises when the 
+    atom shared between the functional groups is a *placer* atom.
+    
+    *Placer* atoms are used to align building blocks, so that
+    they have an appropriate orientation in the final topology.
+    If there is only one *placer* atom, no alignment can be made,
+    as no vector running between *placer* atoms can be defined,
+    and used for the alignment of the :class:`.BuildingBlock`.
+    
+    By default, :mod:`stk` may create overlapping functional
+    groups, which may lead to a lack of an appropriate number
+    of *placer* atoms, leading to a :class:`.BuildingBlock` 
+    being unalinged. However, the user can manually set the 
+    *placer* atoms of functional groups, so that the *placer*
```

Comment:
'...so that not all *placer* atoms...'

did I understand this?

Reply:
yes